src/Stripe.net/Services/PaymentIntents/StripePaymentIntentConfirmOptions.cs

```diff
@@ -0,0 +1,17 @@
+using System.Collections.Generic;
+using Newtonsoft.Json;
+
+namespace Stripe
+{
+    public class StripePaymentIntentConfirmOptions : StripeBaseOptions
+    {
```

Comment:
Those are a bit frustrating, they each support a subset of the other endpoints params. Not sure this was the best approach